content/en/users/data/management/datahub/file-management/_index.md

```diff
@@ -102,3 +102,95 @@ In order to edit ACLs:
 The order in which permissions take precedence is indicated with an arrow.
 
 ![DataHub ACL](datahub-acl.png)
+
+> To limit the risk of making data inaccessible to yourself it is advisable to
+> first add the required ACL if new ones are needed and if necessary remove the
+> ones not needed. While using ACLs there should be at least one active or no
+> user might be able to access the data. To regain the access the space owner
+> would then need to modify the permissions.
+
+## File sharing
+
+It is possible to share DataHub data available with other user of DataHub or
+external by generating a unique URL. This can be done, on the web interface by
+selecting a directory or file and right-clicking on it or clicking the three
+dots on the right like in the following screenshot and selecting "share":
+
+![select](datahub-share-01.png)
+
+this will open a window that allows you to give a name to the share that you are
+about to create. This is shown in the following screenshot:
```

Comment:
```suggestion
This will open a window allowing you to give a name to the share that you are
about to create. This is shown in the following screenshot:
```